lib/src/parsers/headers-parser.spec.ts

```diff
@@ -45,12 +47,40 @@ describe("headers parser", () => {
     });
     expect(result[2]).toStrictEqual({
       description: "property description",
+      examples: undefined,
       name: "property-with-description",
       type: {
         kind: TypeKind.STRING
       },
       optional: false
     });
+    expect(result[3]).toStrictEqual({
+      description: "property-example description",
+      examples: [{ name: "property-example", value: "property-example-value" }],
+      name: "property-with-example",
+      type: {
+        kind: TypeKind.STRING
+      },
+      optional: false
+    });
+    expect(result[4]).toStrictEqual({
+      description: "property-two-examples description",
+      examples: [
+        {
+          name: "property-example-one",
+          value: 123
```

Comment:
Given we are storing different types for `value`, can we update the corresponding definition to `value: any`:

```ts
// definitions.ts
export interface Example {
  name: string;
  value: any;
}
```

Reply:
Should this be either a generic or `string | integer | boolean`?

Reply:
I think this can be left as is until we need to encapsulate type properly. At that point I think we can replace it with some subset of `Type` or `TypeDef`.